### PR TITLE
Adapt to centos stream

### PR DESCRIPTION
**Pull request by snir911**
Tested and with CentOS Stream 8

Fixes: #4299

### CODE REVIEW

**Review by Jakob-Naucke — COMMENTED** [review]
@snir911 you're Red Hat and I'm not, but I think the correct branding is Fedora (uppercase) :)

File: .ci/setup_env_centos.sh
Comment (wainersm) [inline .ci/setup_env_centos.sh:29]: I wonder if that block is really needed for CentOS Stream at all. I bet there are many tweaks carried on the `setup_env_centos.sh` not needed, so I suggest we start with a almost fresh script.

Comment (snir911) [reply .ci/setup_env_centos.sh:29]: TBH i thought about it but i was lazy and assumes (maybe wrongly) CentOS stream is closer to Centos than Fedora so it's needed 
But wouldn't it make sense to drop CentOS in favor of CentOS-Stream later and just use the same script (dropping what's not needed)?

File: .ci/README.md
Comment (jodh-intel) [inline .ci/README.md:124]: Why `wget` and not `curl`?

File: .ci/README.md
Comment (jodh-intel) [inline .ci/README.md:134]: I appreciate this is just updating what was already there, but a couple of comments:

- There are lots of commands here, so why not put them into a script and just say "run this script"? That way the script could perform additional error checking, etc.
- Again, I know this isn't strictly part of the PR but these commands should have a shell prompt prefix (see https://github.com/kata-containers/kata-containers/blob/main/docs/Documentation-Requirements.md#code-blocks).

Comment (snir911) [reply .ci/README.md:134]: Actually you can run it as script on all platforms, if the pkg is missing it will be ignored and Cgroupsv1 is already present for centos (currently) so it won't hurt, do you think it still worth condition those?

Oh, there is `"'sh` but it seems it's not interpreted as `'"bash`, I'll switch
Thanks!

File: .ci/README.md
Comment (snir911) [reply .ci/README.md:124]: As it was copied i supposed it's used by the scripts somewhere, I'll try to remove tit and see if it's actually required

Comment (jodh-intel) [reply .ci/README.md:124]: Thanks!

**Comment by jodh-intel** [comment]
Any update on this @snir911?

**Comment by snir911** [comment]
/retest
@jodh-intel i partly fixed and comment on your comment, i think it's fine to have it as is, let me know otherwise

File: .ci/setup_env_centos.sh
Comment (Jakob-Naucke) [reply .ci/setup_env_centos.sh:29]: ^ @wainersm

File: .ci/README.md
Comment (jodh-intel) [inline .ci/README.md:134]: You could prefix to remove the comment:

```bash
[ $(source /etc/os-release && echo $ID) = fedora ] && sudo grubby ...
```

File: .ci/README.md
Comment (jodh-intel) [inline .ci/README.md:124]: All command that users type are supposed to have a `$ ` prompt on them.

File: .ci/README.md
Comment (snir911) [reply .ci/README.md:134]: cool, I assumes it would be more acute to check which cgroups is used, however i don't have a good way of doing that
best i have so far:
`! mount | grep "/sys/fs/cgroup\ " | grep cgroup2`
or
`( ! mount -t cgroup2 ) || ( mount -t cgroup2 | grep unified )`

File: .ci/README.md
Comment (snir911) [reply .ci/README.md:124]: will fix

**Comment by fidencio** [comment]
A head's up that may affect your PR.

In the Architecture Committee meeting from January 25th, 2022, the Architecture Committee has agreed on using the "Dismiss stale pull request approvals when new commits are pushed" configuration from GitHub.  It basically means that if your PR has been rebased or updated, the approvals given will be erased.

In order to minimize traumas due to the new approach, please, consider adding a note on the changes done before the rebase / force-push, and also pinging the reviewers for a subsequent round of reviews.

Thanks for your understanding!

Related issue: https://github.com/kata-containers/community/issues/249

File: .ci/README.md
Comment (snir911) [reply .ci/README.md:128]: I think you are right, I just left it as is, maybe it's not true in more legacy fedoras? you think it's safe to remove?

File: .ci/README.md
Comment (snir911) [reply .ci/README.md:133]: Actually the whole code block can be treated as a script, i think the idea was that this way it's easier to pay attention what your're doing and modify lines if needed, maybe i can just change it all to a script if preferred

Comment (Jakob-Naucke) [reply .ci/README.md:133]: No, okay, I get that. Let's keep it.

File: .ci/README.md
Comment (Jakob-Naucke) [reply .ci/README.md:128]: That `sudo -H` has to be good for _something_, right? :p I think it's safe.

**Review by Jakob-Naucke — APPROVED** [review]
No showstoppers here IMO, let's make the CI ready for Stream.

**Review by fidencio — DISMISSED** [review]
lgtm, thanks @snir911!

**Comment by Jakob-Naucke** [comment]
> ```
> time="2022-01-31T13:23:10Z" level=warning msg="Failed Minval (0.510000 > 0.507350) for [boot-times]"
> ```

so that's... good? For now, I'm going to /retest-metrics and cc @GabyCT

File: .ci/README.md
Comment (c3d) [inline .ci/README.md:119]: Is it smart to add CentOS knowing it's going the way of the dodo?

Comment (c3d) [reply .ci/README.md:119]: While you are editing this, why not add the versions that are "known to work"?

Comment (snir911) [reply .ci/README.md:119]: will do

**Review by Jakob-Naucke — APPROVED** [review]

**Review by fidencio — APPROVED** [review]
lgtm, thanks @snir911!

**Pull request merged**
